packages/mdc-ripple/mdc-ripple.scss

```diff
@@ -35,11 +35,11 @@
   }
 
   &--primary {
-    @include mdc-ripple-color(primary, $mdc-ripple-pressed-dark-ink-opacity);
+    @include mdc-states(primary, true);
   }
 
   &--accent {
-    @include mdc-ripple-color(secondary, $mdc-ripple-pressed-dark-ink-opacity);
+    @include mdc-states(secondary);
```

Comment:
Why does `--primary` set `$has-nested-focusable-element: true` but `--accent` does not?

Reply:
Because I'm dumb and forgot to remove something I put in simply to verify that the mixin worked as intended (since the only component that will actually use that parameter is text field). :D Fixed.